config/locales/misc/en.yml

```diff
@@ -31,6 +31,7 @@ en:
     application_policy:
       must_be_logged_in: must be logged in
       you_must_be_logged_in: You must be logged in
+      you_have_a_suspended_account: You have a suspended account
```

Comment:
I feel like "Your account is suspended" would be less ambiguous (it might just be me, and feel free to ignore)

Reply:
I like this wording and will be adding it.